packit_service/worker/helpers/build/build_helper.py

```diff
@@ -385,14 +385,24 @@ def get_check_cls(
         chroot: str = None,
         project_event_identifier: Optional[str] = None,
         identifier: Optional[str] = None,
+        package: Optional[str] = None,
+        template: Optional[str] = None,
     ):
+        if project_event_identifier:
+            project_event_identifier = project_event_identifier.replace(":", "-")
+
+        if template is not None:
+            return template.format(
+                job_name=job_name,
+                chroot=chroot,
```

Comment:
Can we also include a compose name for tests?

Reply:
I have a feeling that chroot is the compose for testing farm :eyes: